docs/faq.md

```diff
@@ -55,3 +55,14 @@ We create a process spec linked to the thermal evaporation process template and
 One ingredient points to "A" as its `material` and has `layer 1` as its name.
 One points to "B" as its `material` and has `layer 2` as its name.
 And the final ingredient spec _also_ points to "A" as its `material` but is differentiated because it has `layer 3` as its name.
+
+## What are is the difference between a description and notes?
+
+`description` is a field on both [Attribute Templates](../specification/attribute-templates) and [Object Templates](../specification/object-templates).
+It is used to describe the type of data that a template is intended to constrain; documentation of intended use.
+It is strongly encouraged to document all templates, given both how important they are in constraining data and communicating structure to analysis algorithms, and in how much reuse a template is likely to get by a range of users on a given  platform.
```

Comment:
```suggestion
We strongly encourage documenting all templates, given both how important they are in constraining data and communicating structure to analysis algorithms, and that a template is likely to be reused by multiple users.
```
I removed "on a given platform" because I imagine that for the typical consumer of this document (an end user), they only see one platform (their deployment)

Reply:
I had platform in there intentionally, since it might not even be a citrine platform -- model independent of the instantiation.  But it's a fair critique, and I'll stick with your change if that's a strong feeling on your part.

Reply:
That is true, but I still think that the important information is conveyed without the clause.